toggl/reports.py

```diff
@@ -16,31 +18,36 @@ def parse_val(v):
             else:
                 return v
 
-        for k, v in attribs.iteritems():
+        try:
+            iteritems = dict.items # Python 3
+        except AttributeError:
+            iteritems = dict.iteritems # Python 2
+
+        for k, v in iteritems(attribs):
             if isinstance(v, list):
                 attribs[k] = [parse_val(el) for el in v]
             elif isinstance(v, dict):
                 attribs[k] = parse_val(v)
         self.__dict__.update(attribs)
 
-    @property
-    def keys(self):
-        return self.__dict__.keys()
-
-    def __unicode__(self):
-        if hasattr(self, '_unicode_value'):
-            if self._unicode_value is None:
-                return '(none)'
+    if sys.version_info[0] < 3:
+        @property
+        def keys(self):
+            return self.__dict__.keys()
+
+    if sys.version_info[0] < 3:
+        def __unicode__(self):
```

Comment:
Maybe it'd be cleaner if `six` (a 3rd-party package helping writing python 2&3 compatible code) were used here to check for string types.